client/src/screens/SignIn/SignIn.jsx

```diff
@@ -78,15 +79,17 @@ export const SignIn = () => {
             type="password"
             value={loginInfo['password']}
             onFocus={() => setAuthError(null)}
+            autoComplete="current-password"
           />
         </div>
         <div className="form-error">
           <h6>{authError}</h6>
         </div>
-        <SingleActionButton text="Log In" type="submit" />
+        <div className="action-btn-container">
+          <SingleActionButton text="Sign In" type="submit" />
```

Comment:
Let's pretend the SingleActionButton is ready with the appropriate payload and props so that when it does get merged, we don't have to come back and update this

Reply:
My thoughts exactly.